.gitmodules

```diff
@@ -32,9 +32,6 @@
 [submodule "plugins/blocks/makeSubmission"]
 	path = plugins/blocks/makeSubmission
 	url = https://github.com/pkp/makeSubmission.git
-[submodule "plugins/reports/reviewReport"]
-	path = plugins/reports/reviewReport
-	url = https://github.com/pkp/reviewReport
```

Comment:
Just double-checking that this was an intended change. I know there were some changes to `reviewReport` so is this not intended to be shipped with core now?

Reply:
Yes, this was just to get past an issue with that plugin in the automated testing; there's a big warning for me to revert this part before merging [over here](https://github.com/pkp/pkp-lib/issues/6264#issuecomment-702926627) (we'll see if I remember it).